### ISSUE TITLE
Add "New Issue" (error tracking) monitor for edx-edxapp-lms

### ISSUE THREAD

**Issue by robrap** [edx/edx-arch-experiments#706]
Blocked on:
- [ ] https://github.com/edx/edx-arch-experiments/issues/647

Acceptance Criteria:
- [ ] Create "New Issue" alert for code_owner_squad arch-bom for edx-edxapp-lms
- [ ] Create Safety Net "New Issue" alert
- [ ] Create or update runbooks
- [ ] Update how-tos and communicate as appropriate

Notes:
-  See https://app.datadoghq.com/monitors/create/error-tracking, "New Issue" alert.

**Comment by robrap** [comment]
@robrap: TODO: Add datadog wiki links

**Comment by robrap** [comment]
For code owner:
```
@code_owner:(arch-bom OR -*)
```

**Comment by robrap** [comment]
@srathod-apphelix: These alerts are noisy in Slack:
- https://twou.slack.com/archives/C04A10SLPFZ.
- We _might_ want to temporarily mute.
- We need to determine how and if we can tune, or whether this just would alert us to too many false negatives. I have not looked into any of the alerts.

**Comment by robrap** [comment]
@srathod-apphelix: Here are some additional notes:
1. I marked as Resolved in DD, and got a ton of Recovered alerts in the Slack channel. like this one: https://twou.slack.com/archives/C04A10SLPFZ/p1748543687367449
2. Here is one of the resolved errors that are not showing the error information in the alert output: https://app.datadoghq.com/error-tracking?query=env%3A%28prod%20OR%20edge%29%20service%3Aedx-edxapp-lms&link_source=monitor_notif&monitor_id=173301327&monitor_sub_type=.new%28%29&source=all&sp=%5B%7B%22p%22%3A%7B%22issueId%22%3A%22bac29ed0-d916-11ef-8a93-da7ad0900002%22%7D%2C%22i%22%3A%22error-tracking-issue%22%7D%5D&from_ts=1748457190000&to_ts=1748543590000&live=false
3. I had no idea how noisy this would be. If and when we experiment with this again, instead of the Slack channel, you should either not use notification (and just use the DD UI), or notify your email so you can filter it into a folder, until we have it well tuned.

**Comment by jristau1984** [comment]
https://2u-internal.atlassian.net/browse/BOM2-29
